mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```diff
@@ -73,6 +90,15 @@
                     "minItems": 2,
                     "maxItems": 2
                 },
+                "quantization": {
+                    "type": "number"
```

Comment:
I guess it's only an "int" for "integer" types:
https://automl.github.io/ConfigSpace/main/api/hyperparameters.html#module-ConfigSpace.api.types.integer